doc-content/enterprise-only/admin-and-config/environment-mode-con.adoc

```diff
@@ -0,0 +1,10 @@
+[id='environment-mode-con']
+= Environment mode in {KIE_SERVER} and {CENTRAL}
```

Comment:
I suggest consolidating these three sections into one procedure, "Configuring the environment mode in {KIE_SERVER} and {CENTRAL}". Reason is that this will be reused in several stories, and will be best to economize and just put in the conceptual info as the first paragraph(s) of the proc. This will simplify things so that we don't have to include multiple modules each time reused. Not that that's a problem of course, but just that I don't think it's necessary. And I would say it's not necessary too to split them up in to two procs, one for KIE Server and one for Central, because they're both related.

Take another look at my content in the [7.3 release notes](https://access.redhat.com/documentation/en-us/red_hat_decision_manager/7.3/html/release_notes_for_red_hat_decision_manager_7.3/rn-whats-new-con#new_development_mode_and_project_deployment_enhancements_for_decision_server), and base the concept paragraph(s) on the first two paragraphs that I show there, and use the last paragraph as the basis for the proc. And avoid saying things like "formerly the default setting" and stuff about previous releases in this section. I say that in the RN because of its nature, but we usually don't have that language in typical docs.

The main things we want to cover are configuring the setting in KIE Server, configuring the deployment behavior accordingly in Central (dev mode ON or OFF if KIE Server is in dev mode itself, or OFF only if KIE Server is in prod mode itself), and covering those deployment behaviors and options, i.e., Build vs. Build and Install, Deploy vs. Redeploy, etc. I cover these things pretty superficially in my release note, so we can really just turn that into a proc module, imo.

Does this make sense? If you'd like to talk through it a bit more, ping me and we can chat. Maybe bust through everything else I've mentioned on this PR and circle back to this at the end and we can discuss if needed.

Reply:
I consolidated these, corrected the phrasing, but I am not sure where the enhancements to the build options should go. Building and deploying projects does not appear to be a part of this doc.

Reply:
Finished the further changes and the suggestions I missed. Waiting on Tomas and final review.